tasker_tutorail/c0x_env_vars_backup/Taskfile.yml

```diff
@@ -0,0 +1,78 @@
+# # https://taskfile.dev
+# When doing interpolation of variables, Task will look for the below. They are listed below in order of importance (i.e. most important first):
+
+# Variables declared in the task definition
+# Variables given while calling a task from another (See Calling another task above)
+# Variables of the included Taskfile (when the task is included)
+# Variables of the inclusion of the Taskfile (when the task is included)
+# Global variables (those declared in the vars: option in the Taskfile)
+# Environment variables
+
+version: '3'
+
+# Vars get pulled from included taskfiles
+includes:
+  greet.tasks:
+    taskfile: greet.tasks.yaml
+
+  greet.tasks.override:
+    taskfile: greet.tasks.yaml
+    vars:
+      # but we can override them like this:
+      INCLUSION_GREETING: Hi
+
+vars:
+  GREETING: Here we play with variables
+
+dotenv: ['.env']
+
+tasks:
+  default:
+    desc: Here we use a global variable
+    cmds:
+      - echo "{{.GREETING}}"
+      - task -a
+    silent: true
+
+  greet-env:
+    desc: global variable and enviroment variable do not fight, we can even use them together (not recommended though, it quickly gets complex)
+    cmds:
+      - echo "{{.GREETING}}"
+      - echo $GREETING
+    env:
+      GREETING: Hello, dear user! Here is the environment variable, it doesn't override the global task variable
+
+  greet-var:
+    desc: local variable overrides the global one
+    cmds:
+      - echo "{{.GREETING}}"
+    vars:
+      GREETING: Hello, dear user! Note that the local greeting overrides the global one
+
+
+  greet-call-task:
+    desc: here we call other tasks not overriding any variables
+    cmds:
+      - task greet-env
```

Comment:
may be here you can use `task: greet-env`
but may be we need to have whole totorila for it

Reply:
fixed, add a note